### DISCUSSION TITLE
Formulaire encore présent une fois qu'on a fait "Soumettre"

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#222] [Annonces]
Est-ce qu'un formulaire est toujours existant une fois qu'il a été soumis ? Est-ce que nous pouvons le soumettre de nouveau ?

**Comment by LarcM7** [comment]
Salut! Il existe encore mais FRW reconnaît qu'il est soumis et ne permet pas de le reprendre. Tu ne pourrais donc pas le soumettre à nouveau.

**Comment by ChrisThe01** [comment]
Est-ce qu'il est possible de le récupérer ?

**Reply by LarcM7** [reply]
Pas vraiment!

Toutefois une astuce est de copier le contenu d'un formulaire qu'on souhaite conserver juste avant de le soumettre, à l'aide des [outils de développement](https://github.com/MTESSDev/FRW/blob/main/Documentation/outils-developpement.md#consulter-le-mod%C3%A8le-de-donn%C3%A9es-du-formulaire).

Il est alors possible après d'utiliser le contenu comme pré-remplissage d'un nouveau formulaire.

Vous pouvez ainsi vous constituer une banque de json pour pré-remplir rapidement vos formulaires.

En espérant que cela vous aide. N'hésitez pas s'il y a quoi que ce soit.

Bonne journée!

**Reply by ChrisThe01** [reply]
C'est plutôt en production, Est-ce possible ?

**Reply by LarcM7** [reply]
Ce n'est pas possible de faire quelconque opérations sur un formulaire déjà transmis.

**Comment by ChrisThe01** [comment]
Alors, autre question :
Dans la file de message de la chaine, est-ce qu'on peut relancer la chaine via le console de pilotage ?

**Reply by LarcM7** [reply]
J'assume que vous voulez dire en environnement de développement...
Mais non ce n'est pas une façon de faire qui est supportée. 
Je n'ai aucune idée pourquoi vous voudriez faire ça.
Il me semble que ça risque d'engendrer des erreurs d'intégrité.
